the-collab-lab/tcl-76-smart-shopping-list
# update: tab in browser updated

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

- Updated `index.html` and changed title of app to SnapShop
- Imported new png for tab logo

## Related Issue

closes 53

## Acceptance Criteria

Acceptance Criteria

- [X]  Icon is relevant to SnapShop as a brand
- [X]  Tab reads "SnapShop" instead of "Smart Shopping List"

## Type of Changes

`enhancement`

## Updates

### Before


![](https://files.slack.com/files-pri/TLU3M0S22-F07R3JJQKEC/screen_shot_2024-10-04_at_8.54.50_am.png)

### After

![](https://files.slack.com/files-pri/TLU3M0S22-F07Q0E997HV/screen_shot_2024-10-04_at_9.24.25_am.png)

## Testing Steps / QA Criteria

- Pull from `ar-tab-updates-issue-27`
- See the tab in browser
